TACOCAT.py

```diff
@@ -26,6 +26,10 @@
 #----------------------------------------------------------------------------------#
 ## General Core Information
 
+def test(x,y):

```

Comment:
What is this test here?

Reply:
I added this when I was first resolving a importing issue and forgot to remove it. It should be removed now